src/main.rs

```diff
@@ -375,16 +387,16 @@ fn decorate_track(
             Some("Track") => {
                 decorated = true;
 
+                let mut flags = TriMeshFlags::default();
+                flags.set(TriMeshFlags::FIX_INTERNAL_EDGES, true);
+                let (vtx, idx) =
+                    extract_mesh_vertices_indices(meshes.get(mesh_handle).unwrap()).unwrap();
+                let collider: Collider = SharedShape::trimesh_with_flags(vtx, idx, flags).into();
+
                 commands
                     .entity(mesh_entity)
-                    .insert(ColliderDebugColor(Color::GREEN))
-                    .insert(
-                        Collider::from_bevy_mesh(
-                            meshes.get(mesh_handle).unwrap(),
-                            &ComputedColliderShape::TriMesh,
```

Comment:
Would this have been possible? @rparrett 

```rs
&ComputedColliderShape::TriMesh(TriMeshFlags::FIX_INTERNAL_EDGES),
```

Reply:
[`ComputedColliderShape::TriMesh`](https://docs.rs/bevy_rapier3d/latest/bevy_rapier3d/geometry/enum.ComputedColliderShape.html) doesn't seem to contain any data.

Reply:
Ah, I see, this was added very recently.

Reply:
oh ok my bad thanks for checking it out

Reply:
No worries, appreciate you diving in to help.